static/src/javascripts/projects/common/modules/commercial/adfree/update-user-features.js

```diff
@@ -0,0 +1,33 @@
+define([
```

Comment:
I feel like this doesn't need to be separate module. It just checks expiry date and to the ajax call which can be an 'update' method in 'user-features' module. I like how it is done but in theory merging it into one module removes some work for parser and maintenance.